.travis.yml

```diff
@@ -1,20 +1,26 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+sudo: false
 
 matrix:
+  include:
+    - php: 5.4
```

Comment:
I have some servers running which are stil 5.4, so I think I will keep 5.4 for now